WEAPON/SetWeaponDamageModifierThisFrame.md

```diff
@@ -19,9 +19,6 @@ Changes the weapon damage output by the given multiplier value. Must be run ever
 
 ```lua
 Citizen.CreateThread(function()
```

Comment:
Creating threads shouldn't be necessary if you're not planning on running loops that require delays.